frontend/src/components/phones/onboarding/RealPhoneSetup.tsx

```diff
@@ -120,17 +119,26 @@ const RealPhoneForm = (props: RealPhoneFormProps) => {
       <form onSubmit={onSubmit} className={styles.form}>
         {renderErrorMessage}
 
-        <input
-          className={`${styles["c-verify-phone-input"]}`}
+        <PhoneInput
+          className={`${styles["PhoneInput"]} ${
+            phoneNumberError ? "is-error" : ""
+          }`}
           placeholder={l10n.getString(
             "phone-onboarding-step2-input-placeholder"
           )}
-          ref={phoneNumberField}
+          countries={["US", "CA"]}
```

Comment:
@Vinnl @groovecoder I have not tested using a number for another country.

Reply:
I don't think numbers from other countries are supposed to work... With that in mind, would it be possible to remove the "International" option from the drop-down?